### ISSUE TITLE
persistent markers

### ISSUE THREAD

**Issue by itmitica** [lite-xl/lite-xl-plugins#382]
Is this feature available?

So far, from my experience, when the project is closed/reloaded the markers are all gone.

**Comment by Guldoman** [comment]
Yeah, at the moment there is no simple generic way for plugins to store configs.
We're working on it (lite-xl/lite-xl#1738).
